react-main/src/app/screens/Game/components/Board/components/Square/index.js

```diff
@@ -1,9 +1,11 @@
-import React, { Component } from 'react';
+import React from 'react';
 
-class Square extends Component {
-  render() {
-    return <button className="square">{/* TODO */}</button>;
-  }
-}
+const Square = props => {
```

Comment:
We use classic functions for stateless components. Avoid arrow functions in cases like this.